scripts/kurtosis/network_params.json

```diff
@@ -72,13 +109,12 @@
         "mev_boost_image": "flashbots/mev-boost:1.6",
         "mev_builder_image": "flashbots/builder:latest",
```

Comment:
@piwonskp Shall we specify somehow Lighthouse version for this builder as well?

Reply:
I don't think it matters. It's latest by default: https://github.com/kurtosis-tech/ethereum-package/blob/main/src/package_io/input_parser.star#L443. I don't think it'll ever change in ethereum-package. If it does I guess it'll be for a good reason

Reply:
We could even ommit it in dev & baseline builder if we use latest imo. If ethereum-package changes provider it'll be automatically updated then

Reply:
Yeah so we can simplify if latest is default :)

Reply:
Done